### PR TITLE
Updated focus screen

### LINKED ISSUE #347: Update the Focus Mode Animation
**Actual Behaviour.**  
The current focus mode animation needs to be changes.

**Expected Behaviour**  
Implementation needs to be similar of that of desktop.
Reference: 
https://giphy.com/gifs/9MIlg66ykFVcW2HZBM

**Steps to reproduce it**  

**Additional context**  
> Make it similar to desktop

**Would you like to work on the issue?**
@CoderVishalSehgal

### PR DESCRIPTION
**Pull request by vishal-sehgal**
Fixes #347  

**Changes**:
Implemented the Focus screen as per the Neurolab desktop app.

**Screenshot/s for the changes**: 
Video Demo Link:

**Checklist**: 
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications are done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

**APK for testing**: 
https://drive.google.com/open?id=1fFlIhSucVv5yB7uvf08ndGSaqrecvT9h

### CODE REVIEW

**Comment by vishal-sehgal** [comment]
@jddeep Please have a look here.

**Review by jddeep — CHANGES_REQUESTED** [review]
We'll need to make some changes.

File: app/src/main/java/io/neurolab/fragments/FocusVisualFragment.java [hunk 1]
```diff
@@ -53,23 +67,32 @@ public View onCreateView(@NonNull LayoutInflater inflater, ViewGroup container,
...
-        if (getArguments() != null) {
-            view.findViewById(R.id.animated_view).setVisibility(View.VISIBLE);
-            SpaceAnimationVisuals.spaceAnim(view);
+        rocketAnimation = new SpaceAnimationVisuals(view);
+
+        if (getArguments() != null && StatisticsFragment.parsedData == null) {
+            rocketAnimation.playRocketAnim(view);
             recordState = true;
             filePath = getArguments().getString(LOG_FILE_KEY);
+
```
Comment (CloudyPadmal) [inline app/src/main/java/io/neurolab/fragments/FocusVisualFragment.java]: Remove extra lines and reformat the code

File: app/src/main/java/io/neurolab/fragments/FocusVisualFragment.java [hunk 2]
```diff
@@ -190,10 +223,74 @@ private void selectCSVFile() {
...
+            StatisticsFragment.parsedData = parsedData = strings;
+
+            getActivity().runOnUiThread(new Runnable() {
+                @Override
+                public void run() {
+                    rocketAnimation.animateRocket(convertToDouble(parsedData), getActivity());
+
+                }
+            });
+
```
Comment (CloudyPadmal) [inline app/src/main/java/io/neurolab/fragments/FocusVisualFragment.java]: Clean up the code

File: app/src/main/res/layout/fragment_focus_visual.xml [hunk 3]
```diff
@@ -7,46 +7,65 @@
...
+        <VideoView
+            android:id="@+id/focus_bg"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent" />
+
+        <View
+            android:id="@+id/scrim_focus_screen"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:background="#A61B1B1B"/>
```
Comment (CloudyPadmal) [inline app/src/main/res/layout/fragment_focus_visual.xml]: Move the hardcoded values to resources

**Review by CloudyPadmal — CHANGES_REQUESTED** [review]

**Comment by CloudyPadmal** [comment]
@CoderVishalSehgal could you please resolve the conflicts? This PR is good to be merged

**Comment by CloudyPadmal** [comment]
@jddeep can you follow up @CoderVishalSehgal and get the conflict files resolved?

**Comment by jddeep** [comment]
@CloudyPadmal Just an update: Actually @CoderVishalSehgal has got his Associate Developer exam this Sunday, so he is preparing for that. He will be active once again here, when he is done with that, i.e, from next week.
But nevertheless, the changes directed in this PR has been directed by me in PR #365 along with many improvements. So this being remaining unmerged should not be a problem as of now.

**Review by jddeep — APPROVED** [review]

**Review by CloudyPadmal — APPROVED** [review]

**Pull request merged**